index.html

```diff
@@ -1,13 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="StyleSheet.css" />
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
+	<link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet" />
+    <title> Portfolio-Microverse </title>
+  </head>
+  <body>
+    <div class="containers">
+      <header class="navbar-container">
+        <a class="logo" href="#"> AlexisJW </a>
+        <nav id="navbar">
+          <div class="About"></div>
+          <div class="My works"></div>
+          <div class="Contact"></div>
+        </nav>
+      </header>
+	  
+     <div class="section-1">  
+      <div class="image-background">
+         <div class="headline">
+            <section>
```

Comment:
- [ OPTIONAL] The main container of the **_headline_** section should be the section tag, not the div tag. This makes it more semantic.

```HTML
 <section class="headline">
            <div>
```